data_pipeline/gmail_data/get_gmail_data.py

```diff
@@ -99,12 +98,10 @@ def iter_link_message_thread_ids(service: Resource, user_id: str) -> Iterable[di
 
 def get_link_message_thread_ids(
         service: Resource,
-        user_id: str,
-        is_end2end_test: bool) -> pd.DataFrame:
+        user_id: str
+) -> pd.DataFrame:
     imported_timestamp = get_current_timestamp()
     messages_iterable = iter_link_message_thread_ids(service, user_id)
-    if is_end2end_test:
```

Comment:
here the condition could still make sense since this was to process less data as part of the end to end test

Reply:
(alternatively this could also be part of the config)

Reply:
As we are using the `gmail test account` we don't need it actually, we could remove it before too. I think I have added this part of code before creating test account. Count of emails in this account is less than 10, sometimes I am sending email for test and deleting them. No emails are coming to this account. What do you think should we keep it still or remove?

Reply:
Okay, looks like we wouldn't benefit much from it.